src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -458,6 +543,7 @@ class Seq2SeqTimeSeriesModelOutput(ModelOutput):
     encoder_last_hidden_state: Optional[torch.FloatTensor] = None
     encoder_hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     encoder_attentions: Optional[Tuple[torch.FloatTensor]] = None
+    loc: Optional[torch.FloatTensor] = None
```

Comment:
just want to point out that we'd like to avoid this as much as possible, as it's a breaking change

but I guess we need `loc`here